src/krux/pages/login.py

```diff
@@ -111,7 +111,7 @@ def load_key_from_manual_input(self):
             [
                 (t("Words"), self.load_key_from_text),
                 (t("Word Numbers"), self.pre_load_key_from_digits),
-                (t("Tiny Seed (Bits)"), self.load_key_from_tiny_seed),
+                ("Tiny Seed (Bits)", self.load_key_from_tiny_seed),
```

Comment:
no translations for "Tiny Seed (Bits)" but translations for "Stackbit 1248"?

Reply:
Wow, nice catch. I missed it! Will commit 😉